ja/head.md

```diff
@@ -0,0 +1,100 @@
+# ヘッドの管理
+
+アセットの挿入と同様に、ヘッドの管理も同じ考えに追従しています。つまり、コンポーネントのライフサイクルのレンダリング `context` に動的にデータを付随させ、そして `template` 内にデータを挿入できるという考えです。
+
+そうするためには、ネストしたコンポーネントの内側で SSR コンテキストへアクセスできる必要があります。単純に `context` を `createApp()` へ渡し、これをルートインスタンスの `$options` で露出させることができます。
+
+```js
+// app.js
+export function createApp (ssrContext) {
+  // ...
+  const app = new Vue({
+    router,
+    store,
+    // this.$root.$options.ssrContext というように、すべての子コンポーネントは this にアクセスできます
+    ssrContext,
+    render: h => h(App)
+  })
+  // ...
+}
+```
+
+これと同様のことが `provide/inject` 経由でも可能ですが、そうすると context が `$root` 上に存在することになるため、インジェクションを解決するコストを避けたほうが良いでしょう。
```

Comment:
> This can also be done via provide/inject, but since we know it's going to be on $root, we can avoid the injection resolution costs.

ここ、自信が無いので、重点的にみていただきたいです。特に後半の

> but since we know it's going to be on $root, we can avoid the injection resolution costs.

が自信ないです。